pkgs/top-level/aliases.nix

```
@@ -1408,6 +1408,7 @@ mapAliases ({
   ### Z ###
 
   zabbix40 = throw "'zabbix40' has been removed as it has reached end of life"; # Added 2024-01-07
+  zed-editor = throw "The version of 'zed-editor' previously in nixos-24.05 is outdated and should not be used. Please install zed from a more recent nixpkgs channel.";
```

```suggestion
  zed-editor = throw "The version of 'zed-editor' previously in nixos-24.05 is outdated and should not be used. Please install it from a more recent nixpkgs channel.";
```

`zed` is a different package I am afraid

---

Indeed, done !